src/config.py

```diff
@@ -33,5 +33,9 @@ class airtable:
 
 wbi = WikibaseIntegrator()
 
+WIKIDATA_MAX_RESULTS_PER_SEARCH: int = int(
+    os.getenv("WIKIDATA_MAX_RESULTS_PER_SEARCH", 50)
```

Comment:
using 50 to match the library's default, but realistically, we probably want just 5, which is what I've set in the public repo var